python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -108,6 +107,10 @@ class ImageDifferenceConfig(pexConfig.Config):
         target=ImagePsfMatchTask,
         doc="Warp and PSF match template to exposure, then subtract",
     )
+    decorrelate = pexConfig.ConfigurableField(
+        target=ImageDecorrelationTask,
+        doc="Decorrelate image difference",
+    )
```

Comment:
Do you actually mean a comment? Or add it to the docstring? I will add a comment.